__tests__/integration/app.test.js

```diff
@@ -20,45 +20,74 @@ describe("GET /api/servercheck", () => {
 			});
 	});
 });
+describe("GET methods", () => {
+	describe("GET /api/topics", () => {
+		test("200: responds with an array of topic objects", () => {
+			return request(app)
+				.get("/api/topics")
+				.expect(200)
+				.then(({ body }) => {
+					const { topics } = body;
 
-describe("GET /api/topics", () => {
-	test("200: responds with an array of topic objects", () => {
-		return request(app)
-			.get("/api/topics")
-			.expect(200)
-			.then(({ body }) => {
-				const { topics } = body;
-
-				expect(topics).toBeInstanceOf(Array);
-				expect(topics).toHaveLength(3);
-			});
-	});
-	test("each object should have properties: slug, description", () => {
-		return request(app)
-			.get("/api/topics")
-			.expect(200)
-			.then(({ body }) => {
-				const { topics } = body;
+					expect(topics).toBeInstanceOf(Array);
+					expect(topics).toHaveLength(3);
+				});
+		});
+		test("each object should have properties: slug, description", () => {
+			return request(app)
+				.get("/api/topics")
+				.expect(200)
+				.then(({ body }) => {
+					const { topics } = body;
 
-				expect(topics).toBeInstanceOf(Array);
-				expect(topics).toHaveLength(3);
+					expect(topics).toBeInstanceOf(Array);
+					expect(topics).toHaveLength(3);
 
-				topics.forEach((topic) => {
-					expect(topic).toMatchObject({
-						slug: expect.any(String),
-						description: expect.any(String),
+					topics.forEach((topic) => {
+						expect(topic).toMatchObject({
+							slug: expect.any(String),
+							description: expect.any(String),
+						});
 					});
 				});
-			});
+		});
 	});
-	describe("Error handling", () => {
-		test("400: /api/notavalidpath", () => {
+	describe("GET /api returns an object with a description of all other avaliable endpoints ", () => {
+		test("200: responds with a JSON object", () => {
+			return request(app)
+				.get("/api")
+				.expect(200)
+				.then(({ body }) => {
+					const { endpoints } = body;
+					expect(endpoints).toBeInstanceOf(Object);
+					expect(endpoints).not.toBeInstanceOf(Array);
+				});
+		});
+		test("each object has properies: description, queries, exampleResponse", () => {
 			return request(app)
-				.get("/api/notavalidpath")
-				.expect(404)
+				.get("/api")
+				.expect(200)
 				.then(({ body }) => {
-					expect(body.msg).toBe("path does not exist");
+					const { endpoints } = body;
+					for (const [key, value] of Object.entries(endpoints)) {
```

Comment:
You could alternatively require the endpoints.json object straight into your test file and expect that it is equal to the response.body (would check the values too)